src/ngraph/op/quantized_convolution.cpp

```diff
@@ -0,0 +1,196 @@
+    , m_window_movement_strides(window_movement_strides)
+    , m_window_dilation_strides(window_dilation_strides)
+    , m_padding_below(padding_below)
+    , m_padding_above(padding_above)
+    , m_data_dilation_strides(data_dilation_strides)
+    , m_output_type(output_type)
+    , m_input_axes(input_axes)
+    , m_filter_axes(filter_axes)
+    , m_output_axes(output_axes)
+{
+    constructor_validate_and_infer_types();
+}
+
+void op::QuantizedConvolution::validate_and_infer_types()
+{
+    enum
+    {
+        INPUT_SCALE = 2,
+        INPUT_ZERO_POINT,
+        FILTER_SCALE,
+        FILTER_ZERO_POINT,
+        OUTPUT_SCALE,
+        OUTPUT_ZERO_POINT
+    };
+
+    NODE_VALIDATION_CHECK(this,
+                          get_input_element_type(INPUT_SCALE).is_real() ||
+                              get_input_element_type(FILTER_SCALE).is_real() ||
+                              get_input_element_type(OUTPUT_SCALE).is_real(),
+                          "Scale must be a floating point number");
+
+    NODE_VALIDATION_CHECK(this,
+                          get_input_element_type(0) == get_input_element_type(INPUT_ZERO_POINT),
+                          "Input Zero point element type (",
+                          get_input_element_type(INPUT_ZERO_POINT),
+                          ") must match input element type (",
+                          get_input_element_type(0),
+                          ")");
+
+    NODE_VALIDATION_CHECK(this,
+                          get_input_element_type(1) == get_input_element_type(FILTER_ZERO_POINT),
+                          "Filter Zero point element type (",
+                          get_input_element_type(FILTER_ZERO_POINT),
+                          ") must match filter element type (",
+                          get_input_element_type(1),
+                          ")");
+
+    // TODO Remove these checks once we support channelwise and vector of scales
+    NODE_VALIDATION_CHECK(this,
+                          shape_size(get_input_shape(2)) == 1 ||
+                              shape_size(get_input_shape(3)) == 1,
+                          "Input scale and input zero point shape must be same and 1");
+
+    NODE_VALIDATION_CHECK(this,
+                          shape_size(get_input_shape(4)) == 1 ||
+                              shape_size(get_input_shape(5)) == 1,
+                          "Filter scale and filter zero point shape must be same and 1");
+
+    NODE_VALIDATION_CHECK(this,
+                          shape_size(get_input_shape(6)) == 1 ||
+                              shape_size(get_input_shape(7)) == 1,
+                          "Output scale and output zero point shape must be same and 1");
+
+    auto input_shape = get_input_shape(0);
+    auto filters_shape = get_input_shape(1);
+
+    if (m_data_dilation_strides.size() == 0)
+    {
+        m_data_dilation_strides = conv_default_strides(this, input_shape, filters_shape);
```

Comment:
These defaults should not be stored in this way. If the input shapes change and we need to revalidate, we need to recompute the default values.

Reply:
Similar comment for the other attributes in the next few `if` statements.